### DISCUSSION TITLE
posts/easy-notifications-from-r/

### DISCUSSION THREAD

**Discussion by giscus** [mvuorre/mvuorre.github.io#14] [giscus]
# posts/easy-notifications-from-r/

How to send notifications from R, or any other CLI, to your phone

https://sometimesir.com/posts/easy-notifications-from-r/

**Comment by jmgirard** [comment]
Great post! At least on Windows, it seems to prefer:
system('curl -d "Woke up after .1 second nap!" ntfy.sh/vuorre-r-notifications')
rather than
system("curl -d 'Woke up after .1 second nap!' ntfy.sh/vuorre-r-notifications")

The latter results in an error and the message 'Woke

**Reply by mvuorre** [reply]
Thanks for the feedback Jeffrey! I had no idea. But I'm not surprised that something breaks on Windows 😉

**Comment by DanChaltiel** [comment]
Great system, thanks, I'm sure I'll use it a lot!
You can also use `httr::POST()` so that it works on any OS:
```r
httr::POST("https://ntfy.sh/vuorre-r-notifications", 
           body="Woke up after .1 second nap!")
```
